t/core/config_etcd.t

```diff
@@ -266,7 +266,6 @@ qr/etcd auth failed/
 etcd auth failed
 etcd auth failed
 etcd auth failed
-etcd auth failed
```

Comment:
Does this test case prove that the cli is reused?

Reply:
yes.